setup.py

```diff
@@ -27,13 +27,12 @@
     url="https://github.com/cda-tum/mqtbench",
     keywords="mqt quantum benchmarking performance testing",
     install_requires=[
-        "matplotlib~=3.5.1",
-        "qiskit~=0.35",
-        "pandas>=1.3.5,<1.5.0",
-        "flask>=2.0.2,<2.2.0",
-        "networkx>=2.7.1,<2.9.0",
-        "pytest~=7.1.1",
-        "pylatexenc~=2.10",
+        "qiskit==0.35",
+        "qiskit[visualization]==0.35.0",
```

Comment:
`qiskit[visualization]==0.35.0` should be be enough (no need to include `qiskit==0.35.0` as well).

Reply:
Thanks for that feedback, I commited the change.